src/utils.js

```diff
@@ -171,6 +171,50 @@ module.exports = {
 	},
 
 	/*
+	 * This will generate a regular expression that will only match input from 1 - maxNum
+	 * Currently only supports lists up to 99
+	 */
+	generateRegExpForNumberedList: function(maxNum) {
+		let regex = '';
+		let singleDigit = '[1-9]';
+		let doubleDigitFullRange = '';
+		let doubleDigitPartialRange = '';
+
+		// check < 1 or > 99
+		if (maxNum < 1 || maxNum > 99) {
+			return;
```

Comment:
I wondered if maybe an exception should be thrown, but maybe not.  The key will be how the calling code handles the response.  Looks like it would need to check for null.  If that's the case, maybe make a comment to that affect in the function header comments.  That way if/when someone goes to use this in the future, there will be a reminder of what to expect.